pkg/apis/postgres-operator.crunchydata.com/v1beta1/pgbackrest_types.go

```diff
@@ -171,6 +171,16 @@ type BackupJobs struct {
 	// More info: https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/
 	// +optional
 	PriorityClassName *string `json:"priorityClassName,omitempty"`
+
+	// Scheduling constraints of pgBackRest backup Job pods.
+	// More info: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node
+	// +optional
+	Affinity *corev1.Affinity `json:"affinity,omitempty"`
+
+	// Tolerations of pgBackRest backup Job pods. Changing this value causes a restart.
```

Comment:
❓ What gets restarted? 🤔 Does the CronJob controller kill running Jobs when a CronJob spec changes?

(Maybe this comment is leftover from something else?)

Reply:
Yeah, we ran into this when we were working on upgrades as well -- we have comments on a few fields about how "changes cause restarts", but I'm not sure where those comments originate or if they are accurate (across our range of supported kube envs).

Reply:
(that said, the notion of a job pod getting restarted seems a little different than the notion of a longer-lived pod getting restarted)

Reply:
I see we're not setting [`concurrencyPolicy`](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/cron-job-v1/#CronJobSpec) and I see that becomes `concurrencyPolicy: Allow` in Kubernetes 1.19.

🔧 I don't think this field will kill any jobs, so let's leave the comment out.

Reply:
What do you think about `PriorityClassName` -- that also has a restart comment, but would that follow the same logic?

Reply:
Yep, totally. These fields don't affect Postgres.